_data/simple-icons.json

```diff
@@ -6351,7 +6355,7 @@
         {
             "title": "Open Containers Initiative",
             "hex": "262261",
-            "source": "https://github.com/opencontainers/artwork/tree/master/oci/icon"
+            "source": "https://github.com/opencontainers/artwork/blob/master/oci/icon/black/oci-icon-black.svg"
```

Comment:
Should be https://github.com/opencontainers/artwork/blob/d8ccfe94471a0236b1d4a3f0f90862c4fe5486ce/oci/icon/black/oci-icon-black.svg

FYI, when you open a file like https://github.com/opencontainers/artwork/blob/master/oci/icon/black/oci-icon-black.svg on Github, you can press the key `y` on your keyboard to have the commit hash in the URL liken mentioned in the [guidelines](commit hash).